docs/src/main/asciidoc/includes/openapi/openapi-generator.adoc

```diff
@@ -235,7 +234,7 @@ You must specify the following options:
 |===
```

Comment:
This table is broken (Required Settings), the last line is missing a cell.

Reply:
Fixed.